string tphsmm_name
int8 input_dofs #Number of via_pts
float32[] input
